x-pack/solutions/security/plugins/security_solution/public/asset_inventory/routes.tsx

```diff
@@ -47,8 +47,8 @@ export const AssetInventoryRoutes = () => {
         <SecurityRoutePageWrapper pageName={SecurityPageName.assetInventory}>
           <DataViewContext.Provider value={dataViewContextValue}>
             <SecuritySolutionPageWrapper noPadding>
-              <Suspense fallback={<EuiLoadingSpinner />}>
-                <AllAssetsLazy />
+              <Suspense fallback={<Loading />}>
```

Comment:
Since this loading component include the page title in it, i would recommend to rename it as well